ARCharts/ARBarChart.swift

```diff
@@ -23,20 +23,18 @@ public class ARBarChart: SCNNode {
         self.geometry = geometry
     }
     
-    
-    /// Initialize an `ARBarChart` with a dataSource and bounding dimensions.
-    /// Dimensions should be read as if you were looking down on the graph from above.
-    /// Width and height determine the square in the XY-plane on which the graph lies.
-    /// Depth is the distance from the planar surface to the top of the bar in the graph.
-    ///
-    /// - Parameters:
-    ///   - dataSource: Provides data to the chart.
-    ///   - width: Distance taken up by x-dimension (columns).
-    ///   - height: Distance taken up by y-dimension (rows).
-    ///   - depth: Distance taken up by z-dimension (from surface to top of largest bar).
-    public init(dataSource: ARBarChartDataSource, width: CGFloat, height: CGFloat, depth: CGFloat) {
+    /** Initialize an `ARBarChart` with a dataSource and bounding dimensions.
+     * Dimensions should be read as if you were looking down on the graph from above.
+     * Width and height determine the square in the XY-plane on which the graph lies.
+     * Depth is the distance from the planar surface to the top of the bar in the graph.
+     *
+     *  - parameter dataSource: Provides data to the chart.
+     *  - parameter size: The size of the graph.
+     */
+    public init(dataSource: ARBarChartDataSource, delegate: ARBarChartDelegate, size: SCNVector3) {
```

Comment:
OK nice, the triple-slash gets auto-generated from holding ctrl-alt-/ for me, but good to stay consistent.